### PR TITLE
Add a close button to the Snippets pane

### PR DESCRIPTION
**Pull request by zadjii-msft**
As discussed in the bug bash. It should be closable with a button. 

This also changes the tab color to match the Settings tabs.

This also fixes a crash where dragging just a snippets pane out to it's own window would crash.

### CODE REVIEW

File: src/cascadia/TerminalApp/SnippetsPaneContent.xaml [hunk 1]
```diff
@@ -225,11 +225,22 @@
...
-                   Margin="4"
-                   FontSize="24" />
+        <Grid Grid.Row="0"
+              Margin="4"
+              HorizontalAlignment="Stretch">
+            <TextBlock x:Name="_title"
+                       x:Uid="SnippetPaneTitle"
+                       HorizontalAlignment="Left"
+                       FontSize="24" />
+            <Button x:Uid="CloseSnippetsPaneButton" HorizontalAlignment="Right" Click="_closePaneClick">
```
Comment (DHowett) [inline src/cascadia/TerminalApp/SnippetsPaneContent.xaml:235]:
```suggestion
            <Button x:Uid="CloseSnippetsPaneButton" HorizontalAlignment="Right" Click="Close">
```

does this work?

**Review by DHowett — APPROVED** [review]

**Review by lhecker — APPROVED** [review]
(Preliminary 2nd approve. Same question as Dustin.)

File: src/cascadia/TerminalApp/SnippetsPaneContent.xaml (hunk 1 above)
Comment (zadjii-msft) [reply src/cascadia/TerminalApp/SnippetsPaneContent.xaml:235]: Nope, cause `Close()` doesn't take the two arguments needed for a `Click` handler

Comment (DHowett) [reply src/cascadia/TerminalApp/SnippetsPaneContent.xaml]: I'd thought the XAML compiler was flexible and used metadata to determine the arity of the event handler, but it appears I had mixed it up with something else.

**Review by DHowett — APPROVED** [review]

**Pull request merged**